docs/workflow/building/libraries/webassembly-instructions.md

```diff
@@ -1,39 +1,22 @@
-# Build for WebAssembly
+# Build libraries for WebAssembly
 
 ## Prerequisites
 
 If you haven't already done so, please read [this document](../../README.md#Build_Requirements) to understand the build requirements for your operating system.
 
-The **correct version** of Emscripten SDK (emsdk) needs to be installed.
-* Run `make -C src/mono/wasm provision-wasm` to install emsdk into `src/mono/wasm/emsdk`.
-* Alternatively follow the [installation guide](https://emscripten.org/docs/getting_started/downloads.html#sdk-download-and-install).
-Do not install `latest` but rather specific version e.g. `./emsdk install 2.0.23`. See [emscripten-version.txt](../../../../src/mono/wasm/emscripten-version.txt)
+The **correct version** of Emscripten SDK (emsdk) needs to be installed. To read how to install it, see [Installing emsdk](../../../../src/mono/wasm/README.md#installing-emsdk).
 
-Once installed the `EMSDK_PATH` environment variable needs to be set:
+## Building
 
-On Linux and macOS:
-
-```bash
-export EMSDK_PATH=<FULL_PATH_TO_SDK_INSTALL>/emsdk
-```
-
-## Building everything
-
-At this time no other build dependencies are necessary to start building for WebAssembly.
+At this time no other build dependencies are necessary to start building for WebAssembly. To read how to build on specific platforms, see [Building](../../../../src/mono/wasm/README.md#building).
```

Comment:
"because emscripten would be downloaded and installed automatically"

Reply:
I meant: "because in the previous step we told you to configure emscripten and we assume you've done it". How automatically?

Reply:
I see